convert/src/vertex_to_bd/vertex_to_badgerdoc_use_case.py

```diff
@@ -87,8 +117,131 @@ def upload_badgerdoc_to_s3(
             self.badgerdoc_format.export_annotations(
                 badgerdoc_annotations_path
             )
+            s3_output_annotations_path = self.get_output_annotations_path(importjob_id_created)
             self.s3_client.upload_file(
                 str(badgerdoc_annotations_path),
-                s3_output_annotations.bucket,
-                s3_output_annotations.path,
+                self.s3_output_bucket,
+                s3_output_annotations_path,
             )
+
+    def request_jobs_to_create_importjob(self):
+        post_importjob_url = f"{settings.job_service_url}create_job/"
+        LOGGER.info(f"{post_importjob_url=}")
+        post_importjob_body = {
+            "name": f"import_job_{uuid4()}",
+            "type": "ImportJob",
+            "import_source": self.s3_input_annotation.path,
+            "import_format": "Label Studio",
+            "owners": [
+                self.token_data.user_id,
+            ],
+            "files": [],
+            "categories": [],
+        }
+        LOGGER.info(f"{post_importjob_body=}")
+
+        try:
+            request_to_post_importjob = requests.post(
+                url=post_importjob_url,
+                headers=self.request_headers,
+                json=post_importjob_body,
+                timeout=10,
+            )
+            request_to_post_importjob.raise_for_status()
+        except requests.exceptions.RequestException as e:
+            raise requests.exceptions.RequestException(
+                f"Failed request to 'jobs' - {e}"
+            ) from e
+
+        else:
+            LOGGER.info(f"{request_to_post_importjob.json()=}")
+            return request_to_post_importjob.json()["id"]
+
+    def request_annotations_to_post_categories(self):
+        post_categories_url = f"{settings.annotation_service_url}categories/"
+        LOGGER.info(f"{post_categories_url=}")
+
+        pages_objs = self.badgerdoc_format.badgerdoc_annotation.pages[0].objs
+        categories = set([pages_obj.category for pages_obj in pages_objs])
+
+        for category in categories:
+            post_categories_body = {
+                "name": category,
+                "type": "box",
+                "id": category,
+            }
+            LOGGER.info(f"{post_categories_body=}")
+
+            try:
+                request_to_post_categories = requests.post(
+                    url=post_categories_url,
+                    headers=self.request_headers,
+                    json=post_categories_body,
+                    timeout=10,
```

Comment:
I would use const or config parameter

Reply:
I've decided to delete 'timeout' argument and go with default value